README.md

```diff
@@ -32,7 +38,7 @@
 1. Download our private keys from the `subscriptions-private` S3 bucket. If you have the AWS CLI set up you can run:
 
     ```
-    aws s3 cp s3://subscriptions-private/DEV/subscriptions-frontend.conf /etc/gu
+    aws s3 cp s3://subscriptions-private/DEV/subscriptions-frontend.conf /etc/gu  --profile membership
```

Comment:
This command is dependent on the configuration of `~/.aws/credentials`, so setting the profile is optional. Please indicate this in the readme.